doc/docs/master/how-tos/export-data-out-pachyderm/mount-repo-to-local-computer.md

```diff
@@ -0,0 +1,187 @@
+# Mount a Repo to a Local Computer
+
+Sometimes you might need to access files directly in
+a Pachyderm repository without downloading them first
+to your local computer. Pachyderm enables
+you to do so with the `pachctl mount` command. This command
+uses the SSH Filesystem (SSHFS) client and Filesystem
+in Userspace (FUSE) user interface to export a Pachyderm
+File System (PFS) to a Unix computer system. In other words,
+it allows you to mount a Pachyderm repository on your computer.
+
+For example, you have an [OpenCV pipeline](../../../getting_started/beginner_tutorial/#image-processing-with-opencv)
+up and running,
+and you need to edit files in the `images` repository. Let's
+say you want to experiment with brightness and contrast
+settings in `liberty.png`. Then, you want the pipelines
+run for the updated file and see the result in the end.
+If you do not mount the `images` repo, you would have to
+first download the files to your computer, edit them,
+and then put them back to the repository. The `pachctl mount`
+command automates all these steps for you. You can mount just the
+`images` repo or all Pachyderm repositories as directories
+on you machine, edit as needed, and, when done,
+exit the `pachctl mount` command. Upon exiting the `pachctl mount`
+command, Pachyderm uploads all the changes to the corresponding
+repository.
+
+## Avoiding Merge Conflicts
+
+If someone else modifies the files while you are working on them
+locally, their changes will likely be overwritten when you exit
+`pachctl mount`. This happens because your changes are saved to
+the Pachyderm repository only after you interrupt the `pachctl mount`
+command. Therefore, make sure that you do not work on the
+same files while someone else is working on them.
+
+Also, if you have any automated way
+to upload files to a Pachyderm repo, and you are modifying files
+on your local computer you might overwrite these changes with
+your changes, when you terminate the `pachctl mount` command.
+
+## Access Permissions
```

Comment:
This info is all good, but should be woven into a broader section introducing the use case for writable mounts. On it's own as an "access permissions" section doesn't really fit. It's much much more than just permissions, it's a whole major feature and use case (and new ways to shoot yourself in the foot -- with great power comes... users making more stupid mistakes too